images/bazelbuild/Makefile

```diff
@@ -22,7 +22,7 @@ BAZEL_VERSIONS=0.5.2 0.5.4 0.6.1 0.7.0 $(LATEST_BAZEL_VERSION)
 all: build
 
 do_build = \
-	docker build --pull --build-arg BAZEL_VERSION=$(VERSION) -t $(IMG):$(TAG)-$(VERSION) .;\
+	docker build --pull --build-arg BAZEL_VERSION=$(VERSION) --build-arg IMAGE=$(IMG):$(TAG)-$(VERSION) -t $(IMG):$(TAG)-$(VERSION) .;\
```

Comment:
Where does `VERSION` come from?

Reply:
see the build target below, do_build is called for each entry in BAZEL_VERSIONS

Reply:
what I'm doing here is here is passing the the same date-sha-version style tag we're tagging the image with along with the image name in as an build arg so we can bake it into the image as an `ENV IMAGE=`.
We then log this in bootstrap so we can tie build logs to the image revision they were produced in.